package.json

```diff
@@ -28,6 +30,19 @@
       "stylelint-config-recommended-scss"
     ]
   },
+  "jest": {
+    "coverageThreshold": {
+      "global": {
+        "branches": 80,
+        "functions": 80,
+        "lines": 80,
+        "statements": 80
+      }
+    },
+    "collectCoverageFrom": [
+      "!src/index.jsx"
```

Comment:
![Screen Shot 2020-08-03 at 5 41 15 PM](https://user-images.githubusercontent.com/53097494/89237519-a0fb3700-d5b0-11ea-9286-db424be430ef.png)

I did try adding this to my packageJson file. 
    "testPathIgnorePatterns": [
      "src/firebase/config.js",
      "src/firebase/firebase.js",
      "src/firebase/uiConfig.js"
    ]
 but got the above error when I then tried to run test coverage. Is it okay to stick with the ignores on the file level since Create React App doesn't yet support testPathIgnorePatterns ?

Reply:
Well, I read what you had backwards, but it was still a fun learning experience. Got it fixed now. Using direct ignores on the file level and cleaned up the package.json file.